### ISSUE TITLE
Use stat from KTH framework?

### ISSUE THREAD

**Issue by paugier** [snek5000/snek5000-cbox#40]
Could we use the stat module of the KTH framework?

**Comment by akhoubani** [comment]
Yes, we can use the stat module of the KTH framework, but we need to change the `.usr` file. See https://github.com/KTH-Nek5000/KTH_Examples/blob/eac7d10cc1b03278874f2a98a9a691b165dee4f0/pipe_STAT/pipe.usr#L60
